helm.tf

```diff
@@ -60,7 +43,7 @@ resource "helm_release" "cluster_autoscaler" {
   namespace  = "kube-system"
   repository = "https://kubernetes.github.io/autoscaler"
   chart      = "cluster-autoscaler"
-  version    = "9.9.2"
+  version    = "9.16.1"
```

Comment:
We should allow chart version selection (module input)

Reply:
I'm marking this as resolved because we agreed to don't do it